README.md

```diff
@@ -1,62 +1,123 @@
+Soot-j9 is an (experimental) extension of Soot for the java 9 module system.
+For Soot details see http://www.sable.mcgill.ca/soot/.
 
-# What is Soot?
 
-Soot is a Java optimization framework. It provides four intermediate representations for analyzing and transforming Java bytecode:
 
-* Baf: a streamlined representation of bytecode which is simple to manipulate.
-* Jimple: a typed 3-address intermediate representation suitable for optimization.
-* Shimple: an SSA variation of Jimple.
-* Grimp: an aggregated version of Jimple suitable for decompilation and code inspection.
+**Note that Soot-j9 is not yet stable nor complete.**
 
-See http://www.sable.mcgill.ca/soot/ for details.
+# How do I obtain Soot-j9 nightly builds
 
-# How do I get started with Soot?
+Nightly builds of Soot can be obtained from [nightly build](https://soot-build.cs.uni-paderborn.de/nightly/soot/). The "sootj9-trunk.jar" file is an all-in-one file that also contains all the required libraries. 
 
 We have some documentation on Soot in the [wiki](https://github.com/Sable/soot/wiki) and also a large range of [tutorials](http://www.sable.mcgill.ca/soot/tutorial/index.html) on Soot.
 
-# Including Soot in your Project
-
-A Soot SNAPSHOT is currently built for each commit to the `develop` branch. You can include Soot as 
-a dependency via Maven, Gradle, SBT, etc using the following coordinates:
-
-
-```.xml
-<dependencies>
-  <dependency>
-    <groupId>ca.mcgill.sable</groupId>
-    <artifactId>soot</artifactId>
-    <version>3.3.0-SNAPSHOT</version>
-  </dependency>
-</dependencies>
-<repositories>
-  <repository>
-      <id>sonatype-snapshots</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-      <releases>
-          <enabled>false</enabled>
-      </releases>
-  </repository>
-</repositories>	
+
+
+# What works?
+
+ * Automatic modules (modules automatically created from jars in the module-path)
+ * Named modules
+ * Exploded modules
+ * Modular jar files
+ * Resolving modules in Soot's `ModuleScene`
+ * Spark
+
+# What does not work yet?
+ * Anonymous modules (mixing module- and class-path)
+ * Multi-module jar files
+
+
+# How to use Soot-j9?
+
+## Use from Source Code
+To load modules in Soot's `ModuleScene` from java:
+```
+// configure Soot's options
+Options.v().set_prepend_classpath(true); // this is currently required to include the virtual java 9 filesystem jrt://
+Options.v().set_soot_modulepath(modulePath);
+
+
+// load classes from modules into Soot
+  	Map<String, List<String>> map = ModulePathSourceLocator.v().getClassUnderModulePath(modulePath);
+        for (String module : map.keySet()) {
+            for (String klass : map.get(module)) {
+                logger.info("Loaded Class: " + klass + "\n");
+                loadClass(klass, false, module);
+
+            }
+    }
+
+
+    //this must be called after all classes are resolved
```

Comment:
Do you mean "loaded" instead of "resolved"? Pleease fix if so, it might get confusing with the resolving process otherwies.